src/Compilers/Core/MSBuildTask/Microsoft.VisualBasic.Core.targets

```diff
@@ -97,6 +101,7 @@
           PreferredUILang="$(PreferredUILang)"
           References="@(ReferencePath)"
           RemoveIntegerChecks="$(RemoveIntegerChecks)"
+          ReportAnalyzer="$(ReportAnalyzer)"
```

Comment:
I believe there are other .targets files that call the Csc and Vbc tasks, I believe related to how WPF processes XAML files. You will want to update those to pass ReportAnalyzer as well.